### PR TITLE
Add class on user bubble for styling

### PR DESCRIPTION
**Pull request by pmontoya**
I just add a class to user bubble element to allow styling on it

### CODE REVIEW

File: lib/steps_components/text/TextStep.jsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ class TextStep extends Component {
     const showAvatar = user ? !hideUserAvatar : !hideBotAvatar;
 
     return (
-      <TextStepContainer className="rsc-ts" user={user}>
+      <TextStepContainer className={`rsc-ts ${user ? 'rsc-ts-user' : null}`} user={user}>
```
Comment (LucasBassetti) [inline lib/steps_components/text/TextStep.jsx]: please change it to `rsc-ts ${user ? 'rsc-ts-user' : 'rsc-ts-bot'}`

Comment (pmontoya) [reply lib/steps_components/text/TextStep.jsx]: You are right. Done.

Comment (Milos5611) [reply lib/steps_components/text/TextStep.jsx]: HI @LucasBassetti when this one can be merged ?
I need it desperetelly :)

**Comment by Milos5611** [comment]
Hi @LucasBassetti when will you publish this merged PR ?

**Pull request merged**